examples/java/uk/co/real_logic/sbe/examples/ExampleUsingGeneratedStub.java

```diff
@@ -119,7 +119,7 @@ public static int encode(final Car car, final UnsafeBuffer directBuffer, final i
            .modelYear(2013)
            .available(BooleanType.TRUE)
            .code(Model.A)
-           .putVehicleCode(VEHICLE_CODE, srcOffset);
```

Comment:
The example was to show the difference between using Strings and raw byte arrays. The Strings are more usable but introduce garbage. Many in HFT want their applications to be garbage free to avoid the subsequent pauses.